extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php

```diff
@@ -238,9 +238,11 @@ function getBody() {
 	}
 
 	function getPageHeader() {
-		global $wgScript, $wgRequest;
+		global $wgScript;
 		$self = $this->getTitle();
-		$this->getLangs();
+		$this->mTopLanguages = Language::getSupportedLanguages();
+		$this->mLanguages = Language::getRequestSupportedLanguages();
+		asort($this->mLanguages);
 
 		$hubs = WikiFactoryHub::getInstance();
 		// fixme, this logic (and the form) need to be split into Verticals and Categories
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this "FIXME" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1134)